controls/utilities.py

```diff
@@ -68,26 +68,28 @@ def oscalize_control_id(cl_id):
 
     return cl_id
 
-
-def oscalize_catalog_key(catalogkey):
-    """ Covers empty catalog key case. Otherwise, outputs an oscal standard catalog key from various common formats for catalog keys
-    NIST_SP_800_53_rev4 --> NIST_SP-800-53_rev4
+def oscalize_catalog_key(catalogkey=None):
+    """Outputs an oscal standard catalog key from various common formats for catalog keys
+    Covers empty catalog key case
+    Example: NIST_SP_800_53_rev4 --> NIST_SP-800-53_rev4
     """
+    DEFAULT_CATALOG_KEY = 'NIST_SP-800-53_rev5'
+    if catalogkey is None or catalogkey=='':
```

Comment:
Change:
if not catalogkey:
        return DEFAULT_CATALOG_KEY

Reply:
tabs not showing up but you see what i  mean

Reply:
Fixed.